data-raw/healthy-lives/behavioural-risk-factors/alcohol-misuse.R

```diff
@@ -0,0 +1,87 @@
+# Load packages
+library(tidyverse)
+library(readxl)
+library(sf)
+library(httr)
+library(httr)
+library(dplyr)
+
+# URL of the Excel file
```

Comment:
Good use of comments throughout to explain to the reader what you are doing - which is very helpful. 

However comments should, in general, provide additional information / explanation that the code does not rather than explain the code. Please refer to this [guide](http://adv-r.had.co.nz/Style.html) (the comments section) and amend your comments throughout the script.